### PR TITLE
[14.0][FIX] maintenance_plan: fix maintence request form view

### PR DESCRIPTION
**Pull request by kluna1998**

### CODE REVIEW

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
image
without the change, this is how it was viewed.

File: maintenance_plan/views/maintenance_equipment_views.xml [hunk 1]
```diff
@@ -63,25 +63,26 @@
         <field name="name">equipment.request.form.inherit</field>
         <field name="model">maintenance.request</field>
         <field name="priority" eval="999" />
-        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_form" />
+        <field name="inherit_id" ref="base_maintenance.equipment_request_view_form" />
         <field name="arch" type="xml">
             <field name="maintenance_type" position="after">
                 <field name="maintenance_kind_id" />
             </field>
             <!-- TODO: Use position="move" instead -->
```
Comment (LoisRForgeFlow) [inline maintenance_plan/views/maintenance_equipment_views.xml:71]: I guess you can remove this comment now

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**